src/Microsoft.AspNetCore.Razor/CodeGenerators/CSharpCodeWriter.cs

```diff
@@ -319,6 +319,20 @@ public CSharpCodeWriter WriteMethodInvocation(string methodName, bool endLine, p
                 .WriteEndMethodInvocation(endLine);
         }
 
+        public CSharpCodeWriter WriteAutoPropertyDeclaration(string accessibility, string typeName, string name)
```

Comment:
Correct. Do you recommend taking this out? We could maybe start a `MvcCSharpCodeWriter` class, or not have it at all since we only really use it once or twice. Just thought it might be useful for others in the future.